ibodata/lateral_profile.py

```diff
@@ -120,7 +120,24 @@ def normalize(self, dt, allow_cast=True):
         mid = self.x_at_y(a) + w / 2.0
         self.x -= mid
 
-        ave = np.average(self.y[np.fabs(self.x) <= dt])
+        slice_y = self.y[np.fabs(self.x) <= dt]
```

Comment:
Slice usually referers to indices, not to subset of numpy array. See i.e.:

```
>>> l = ['a', 'b', 'c', 'd' ]
>>> l[2:3]
['c']
>>> slice_tmp = slice(2,3)
>>> l[slice_tmp]
['c']
```

I'd recommend to rename this slices to sth like `norm_core_x` or `norm_section_x` (and `_y` also).